src/PatchworkProtocol.sol

```diff
@@ -63,12 +63,18 @@ contract PatchworkProtocol is IPatchworkProtocol, Ownable, ReentrancyGuard {
     /// Current protocol fee configuration
     ProtocolFeeConfig private _protocolFeeConfig;
 
+    /// Proposed protocol fee configuration
+    mapping(string => ProposedProtocolFeeConfig) private _proposedProtocolFeeConfigs;
```

Comment:
naming nit: can we just call these ProposedFeeConfig and _proposedFeeConfigs since it is tracking both protocol and scope fee proposals?